digEmailExtractor/email_extractor.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# @Author: ZwEin
+# @Date:   2016-06-21 12:36:47
+# @Last Modified by:   ZwEin
+# @Last Modified time: 2016-09-30 22:27:59
+
+import copy 
+import types
+from digExtractor.extractor import Extractor
+from EE import EE
+
+class EmailExtractor(Extractor):
+
+    def __init__(self):
+        self.renamed_input_fields = ['text']
+
+    def extract(self, doc):
+        return EE(_output_format='obfuscation').extract_email(doc['text'])
```

Comment:
Will we ever need to change the `_output_format`?  If so, you should add a getter/setter for this extractor.